pkg/services/k8s/apiserver/service.go

```diff
@@ -253,6 +252,8 @@ func (s *service) start(ctx context.Context) error {
 		s.stoppedCh <- prepared.Run(s.stopCh)
 	}()
 
+	<-prepared.MuxAndDiscoveryCompleteSignals()["GrafanaKindDiscoveryReady"]
```

Comment:
this prevents a race condition where core kinds will attempt to register before grafana-apiserver is ready to handle them